src/main/server.js

```diff
@@ -7,25 +7,28 @@ import { Scenario } from './modules/Scenario';
 import { Execution } from './modules/Execution';
 import { Output } from './modules/Output';
 
+const COLECTION_NAME = 'MSM-Scenario';
+
 // Express configuration
 const app = express();
 app.use(json());
-app.use(urlencoded({
-    extended: true
-}));
+app.use(
+    urlencoded({
+        extended: true,
+    })
+);
 
 // DB configuration
 // const serviceAccount = process.env.GOOGLE_APPLICATION_CREDENTIALS;
 admin.initializeApp({
     credential: admin.credential.cert(serviceAccount),
-    databaseURL: 'https://metal-slug-maker-default-rtdb.firebaseio.com'
+    databaseURL: 'https://metal-slug-maker-default-rtdb.firebaseio.com',
```

Comment:
Why you have a comma in here? The lint require it or what? i think it's unnecesary